google/auth/external_account_authorized_user.py

```diff
@@ -297,6 +303,20 @@ def with_token_uri(self, token_uri):
         kwargs.update(token_url=token_uri)
         return self.__class__(**kwargs)
 
+    def with_universe_domain(self, universe_domain):
```

Comment:
Shall we consider to move this method move to a common super class?

Reply:
yes, created a common super class, thank you for your suggestion!